src/OcspBasicResponse.php

```diff
@@ -130,7 +130,12 @@ function ($extension) {
         );
 
         if (isset($filter[0]["extnValue"])) {
-            return $filter[0]["extnValue"];
+            $decoded = ASN1::decodeBER($filter[0]["extnValue"]);
+            if(is_array($decoded)) {
+                return ASN1::asn1map($decoded[0], ['type' => ASN1::TYPE_OCTET_STRING]);
```

Comment:
`asn1map()` can return [false or null](https://github.com/phpseclib/phpseclib/blob/2.0/phpseclib/File/ASN1.php#L540), shouldn't we throw in this case?